charts/victoria-metrics-single/values.yaml

```diff
@@ -47,8 +47,8 @@ server:
   priorityClassName: ""
   # -- Overrides the full name of server component
   fullnameOverride:
-  # -- Data retention period in month
```

Comment:
I think keeping 1->1m as default value is more suitable for most cases.

Reply:
Thanks for your review!

Extra changes in documentation was made because I ran `helm-docs` against `make gen-docs` according to the contribution guide:
- https://github.com/VictoriaMetrics/helm-charts/blob/master/CONTRIBUTING.md

Reply:
Sorry for misleading, I mean keeping 1 which means 1 month, `m` is not a valid unit for retentionPeriod here.

Reply:
Reverted this changes.